src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -252,6 +255,22 @@ public function addRow($row)
         return $this;
     }
 
+    public function appendRow($row)
```

Comment:
How come no public methods in this class have any PHPDoc? This is gonna get confusing when differentiating between `addRow` and `appendRow`